opentracing/tracer.py

```diff
@@ -40,31 +41,84 @@ class Tracer(object):
     def __init__(self):
         self._noop_span_context = SpanContext()
         self._noop_span = Span(tracer=self, context=self._noop_span_context)
+        self._active_span_source = NoopActiveSpanSource()
 
-    def start_span(self,
-                   operation_name=None,
-                   child_of=None,
-                   references=None,
-                   tags=None,
-                   start_time=None):
+    @property
+    def active_span_source(self):
```

Comment:
I am not sure if we want to expose the `active_span_source`, is there any reasonable case when someone would like to be able to access to it directly bypassing the `active_span` method?